src/AlternativesPerformance.h

```diff
@@ -0,0 +1,102 @@
+#ifndef AlternativesPerformance_H
+#define AlternativesPerformance_H
+
+#include "Criteria.h"
+#include "Performance.h"
+#include "PerformanceTable.h"
+#include <iostream>
+#include <iterator>
+#include <map>
+#include <ostream>
+#include <string.h>
+#include <vector>
+
+class AlternativesPerformance : public PerformanceTable {
+public:
+  /**
+   * AlternativesPerformance standard constructor (PerformanceTable surcharged)
+   *
+   * @param perf_vect Vector of performance
+   * @param alt_assignment Map of alternative to the assigned category. Default
+   * alternatives assigned to empty categories
+   */
+  AlternativesPerformance(std::vector<Performance> &perf_vect,
+                          std::map<std::string, std::string> alt_assignment =
+                              std::map<std::string, std::string>{{"", ""}});
+
+  /**
+   * AlternativesPerformance constructor without perf values but set of
+   * criteria to evaluate performance over (PerformanceTable surcharged)
+   *
+   * @param nb_of_perfs Number of performance
+   * @param crits Criteria to evaluate performance over
+   * @param prefix Prefix to use for the name of each Performance created.
```

Comment:
I thing we should maybe use the Category, Categories object here for the map function ?
If we keep it like this, we will never use those types and maybe make the code less readable ?
Open to discuss

Reply:
I think it would be none sense to talk about AA without specifying the category possibilities.
By integrating that, the meta heuristic could be easier to implement ?

Reply:
Do we really want to have Category objects in the hash map? I thought the hash map was just for the ids and from that we could deduce the Category. 
If we change to category then we would also have to change the alternative id to a Performance object to be consistent.